### PR TITLE
Bump mesos version to 1.1.0

### PR DESCRIPTION
**Pull request by ernestas-poskus**
Bumping mesos version

### CODE REVIEW

**Comment by lhoss** [comment]
was 1.1.0 *really* released!?
( at least not yet in the deb/ubuntu repos (as seen from the travis-ci log )

**Comment by ernestas-poskus** [comment]
Strange in is already present in https://github.com/apache/mesos/releases anyway will keep it until it is released.

**Comment by lhoss** [comment]
last week I'm not sure if I looked properly about existing 1.1.0 deb
for sure now I can see it!
```
root@node01:# apt-cache policy mesos
mesos:
  Installed: 1.0.1-2.0.93.ubuntu1404
  Candidate: 1.1.0-2.0.107.ubuntu1404
  Version table:
     1.1.0-2.0.107.ubuntu1404 0
        500 http://repos.mesosphere.com/ubuntu/ trusty/main amd64 Packages
 *** 1.0.1-2.0.93.ubuntu1404 0
        500 http://repos.mesosphere.com/ubuntu/ trusty/main amd64 Packages
        100 /var/lib/dpkg/status
```
ps: Im just not sure if 1.1.0 is the most stable option .. currently decided to first go with 1.0.1

File: defaults/main.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 mesos_install_mode: "master" # {master|slave|master-slave}
-mesos_version: "1.0.1"
+mesos_version: "1.1.0"
 
 # Debian
-mesos_package_version: "2.0.93"
+mesos_package_version: "2.0.107"
```
Comment (ernestas-poskus) [inline defaults/main.yml]: Great news cc @lhoss

**Pull request merged**